packages/client/webapp/package.json

```diff
@@ -14,6 +14,9 @@
 		"test:ct": "playwright test -c playwright-ct.config.ts"
 	},
 	"dependencies": {
+		"@fortawesome/fontawesome-free": "6.2.0",
```

Comment:
also recently ran into this [HeroIcons](https://heroicons.com/) which are MIT. Just and FYI that it may be useful as well, tagging @YohannParis for reference

Reply:
We have settle on IBM Carbon Icons.